src/Components/Reports/Util.js

```diff
@@ -51,11 +51,23 @@ export const generateFilterText = (filters) => {
     const filterSeparatorOnLine = '\n';
     const hasStateFilter = filters?.stateFilter?.length > 0;
     const hasNameFilter =  filters?.hostnameOrId?.length > 0;
+    const hasOsFilter =  filters?.osFilter?.length > 0;
 
-    if (hasStateFilter || hasNameFilter) {
+    if (hasStateFilter || hasNameFilter || hasOsFilter) {
         filterText = `\nFilters applied\n`;
         filterText = hasStateFilter ? filterText.concat(`State: ${filters.stateFilter.toString()}${filterSeparatorOnLine}`) : filterText;
         filterText = hasNameFilter ? filterText.concat(`Name: ${filters.hostnameOrId}`) : filterText;
+        if (hasOsFilter) {
+            let osText = '';
+            while (filters.osFilter.length) {
+                let osValue = filters.osFilter.splice(0, 10);
+                if (osValue) {
+                    osText = osText.concat(`${osValue.toString()}${filterSeparatorOnLine}`);
```

Comment:
Would it be better if we have trailing spaces for values below `Operating System` ?

![Screenshot from 2022-04-21 14-56-48](https://user-images.githubusercontent.com/61837065/164425122-be77367c-69a3-49ce-b2f6-4ab3931002cb.png)